src/GitTabExpansion.ps1

```diff
@@ -21,6 +21,7 @@ $subcommands = @{
         get-url set-url show prune update
         "
     rerere = "clear forget diff remaining status gc"
+    'sparse-checkout' = "list set add reapply disable init"
```

Comment:
It's worth noting that the subcommand `init` is already deprecated. Should we even include it in tab completion? 

https://git-scm.com/docs/git-sparse-checkout#Documentation/git-sparse-checkout.txt-eminitem